Userland/Libraries/LibJS/Heap/Heap.cpp

```diff
@@ -327,4 +333,9 @@ void Heap::undefer_gc(Badge<DeferGC>)
     }
 }
 
+void Heap::add_inverse_root(Cell* cell)
+{
+    m_inverse_roots.append(cell);
+}
```

Comment:
I think "uproot" would be quite fitting for what's happening here :D

something like

```suggestion
void Heap::uproot_cell(Cell* cell)
{
    m_uprooted_cells.append(cell);
}
```